src/views/v2/day/event/featured-image.php

```diff
@@ -9,30 +9,28 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.4
+ * @version TBD
  *
  */
-$event    = $this->get( 'event' );
-$event_id = $event->ID;
 
-if ( ! has_post_thumbnail( $event_id ) ) {
+if ( empty( $event->thumbnail->large->url ) ) {
 	return;
 }
 
 ?>
 <div class="tribe-events-calendar-day__event-featured-image-wrapper tribe-common-g-col">
 	<a
-		href="<?php echo esc_url( tribe_get_event_link( $event_id ) ); ?>"
-		title="<?php echo esc_attr( get_the_title( $event_id ) ); ?>"
+		href="<?php echo esc_url( $event->permalink ) ?>"
+		title="<?php echo esc_attr( $event->post_title ) ?>"
 		rel="bookmark"
 		class="tribe-events-calendar-day__event-featured-image-link"
 	>
 		<div class="tribe-events-calendar-day__event-featured-image tribe-common-c-image tribe-common-c-image--bg">
 			<div
 				class="tribe-common-c-image__bg"
-				style="background-image: url('<?php echo esc_attr( get_the_post_thumbnail_url( $event_id, 'large' ) ); ?>');"
+				style="background-image: url('<?php echo esc_url( $event->thumbnail->large->url ); ?>');"
```

Comment:
Here too, use `full` in place of `large`.
```suggestion
				style="background-image: url('<?php echo esc_url( $event->thumbnail->full->url ); ?>');"
```